biological-and-pharmaceutical-bulletin.csl

```diff
@@ -0,0 +1,249 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Biological and Pharmaceutical Bulletin</title>
+    <title-short>Biol. Pharm. Bull.</title-short>
+    <id>http://www.zotero.org/styles/biological-and-pharmaceutical-bulletin</id>
+    <link href="http://www.zotero.org/styles/biological-and-pharmaceutical-bulletin" rel="self"/>
+    <link href="http://bpb.pharm.or.jp/document/inst_e.pdf" rel="documentation"/>
+    <author>
+      <name>Shoji Takahashi</name>
+      <email>s.takahashi@elsevier.com</email>
+      <uri>http://www.mendeley.com/profiles/shoji-takahashi3/</uri>
+    </author>
+    <category citation-format="numeric"/>
```

Comment:
same here:
`<category field="biology"/>` (and/or `<category field="chemistry"/>`)